### ISSUE TITLE
Dojima preventing use of FAF in the evening

### ISSUE THREAD

**Issue by rudiger-gh** [AnimatedSwine37/Find-a-Friend#23]
_Version: Custom Sub Menu v1.14_
_The standalone version of Find a Friend 1.9 is NOT affected!_

### Issue

**On 08/14 (Evening), I noticed that Dojima was preventing me from leaving the house via FAF.**
In this case, the social link I selected was Chie. When I try, I get the "exerting his authorative influence" message:

![image](https://user-images.githubusercontent.com/53123052/110119673-0831d180-7db4-11eb-9763-40a23c353eb0.png)

However, if I leave manually via the front door or by selecting "Go outside" from the sub menu, there's no issue:

![image](https://user-images.githubusercontent.com/53123052/110119720-197ade00-7db4-11eb-9c12-5218b02eec32.png)

Something particularly funny is that even when you're outside, selecting the FAF option gives you the same message!

![image](https://user-images.githubusercontent.com/53123052/110119881-44fdc880-7db4-11eb-83db-ea82bcc283a1.png)

Please can you take a look into this? I've attached a save here. Others from the same playthrough can be found [in the archive](https://drive.google.com/drive/folders/1gbqQXQJ1LYFQttry0k2yix_eeh3UkDYp):
[08.14, Sunday - Evening.zip](https://github.com/AnimatedSwine37/Find-a-Friend/files/6090654/08.14.Sunday.-.Evening.zip)

Since this only affects the Custom Sub Menu version, maybe this is one for Tekka?

Thanks!

**Comment by TekkaGB** [comment]
Think this happened after your [pull request](https://github.com/ShrineFox/Persona-4-Golden-Mod-Menu/pull/17/commits/1b79ffad4007412b31a5ab4fdf45739ece28ad32).  The if conditions are completely the same so I'm assuming the version rudiger was testing of yours wasn't up to date with the repo.
- [Your repo's if condition](https://github.com/AnimatedSwine37/Find-a-Friend/blob/4d64171358ebe00c701b17c3f5c56d8281e5c22e/fieldhook.flow#L337)
- [Custom Sub Menu Repo's if condition](https://github.com/ShrineFox/Persona-4-Golden-Mod-Menu/blob/d2079baa02fba6fccdb5f1bb350add5911748af7/OtherMods/FindAFriend.flow#L269)

**Comment by rudiger-gh** [comment]
To confirm:
 I just tested again by doing a fresh reinstall of Custom Sub Menu (v1.14) from Gamebanana and the issue persists. Can you recreate this with the test save provided above @TekkaGB ?

**Comment by AnimatedSwine37** [comment]
Sorry I've only just now had time to look at this. 

What I've found is that the v1.9 of Find A Friend that I released (stand alone) doesn't contain the Dojima warning which is why it's only popping up in Custom Sub Menu (because that actually does have it). 

I have no idea how I accomplished this but I did, so don't worry about fixing it @TekkaGB, this is a problem with FAF that I'll have to fix. I'll make a pr when I've done so.
